Change list-headers to view the chainheaders instead

SWvheerden opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->
The list-headers command used to display the headers. Some of the detail was removed from the headers such as the target_difficulty.  This PR will now displays the chain_header which still has this detail in it. 
Adds in an option to printout in csv, some raw header deatails 

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [ ] Documentation

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
* [x] I'm merging against the `development` branch.
* [x] I ran `cargo-fmt --all` before pushing.
* [x] I ran `cargo test` successfully before submitting my PR.
* [x] I have squashed my commits into a single commit.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

delta1 commented on base_layer/core/src/chain_storage/accumulated_data.rs:
```diff
@@ -278,12 +279,40 @@ pub struct BlockHeaderAccumulatedData {
+        )?;
+        writeln!(
+            f,
+            "Accumulated monero difficulty: {}",
```
Duplicate “Monero”?

delta1 commented on base_layer/core/src/chain_storage/accumulated_data.rs:
```diff
@@ -278,12 +279,40 @@ pub struct BlockHeaderAccumulatedData {
+        writeln!(
+            f,
+            "Accumulated monero difficulty: {}",
+            self.accumulated_blake_difficulty
```
Should this be SHA instead of Blake?

SWvheerden replied:
No, its a legacy variable name. I believe refactoring this is beyond the scope of this PR.

delta1 replied:
Oh yeah, no problem.

stringhandler commented:
Looks fine, clippy CI failed though

SWvheerden committed:
Change list-headers to view chain_headers instead

Update accumulated_data.rs

Add header stats parser function to get easy to manipulate header data
